projects/pcre2/build.sh

```diff
@@ -21,6 +21,8 @@
 make -j$(nproc) clean
 make -j$(nproc) all
 
+cp $SRC/pcre2_fuzzer.dict $SRC/pcre2_fuzzer.options $OUT/
```

Comment:
I moved the line to the end of the script, and seem to have created a new pull request. The .options file was only committed to the PCRE2 SVN repo today, so perhaps the github repo hasn't pulled it yet.